shared/src/email/emailTemplatesByName.ts

```diff
@@ -1067,8 +1067,44 @@ Ne tardez pas : répondez lui directement en utilisant le bouton ci-dessous : `,
         subContent: defaultSignature(internshipKind),
       }),
     },
+    AGENCY_ASSESSMENT_NOTIFICATION: {
+      niceName: "Prescripteurs - Lien de création du bilan",
+      tags: ["notification prescripteur fin de l’immersion"],
+      createEmailVariables: ({
+        agencyLogoUrl,
+        assessmentCreationLink,
+        beneficiaryFirstName,
+        beneficiaryLastName,
+        businessName,
+        conventionId,
+        internshipKind,
+      }) => ({
+        subject: `Immersion Facilitée - Bilan non complété pour l'immersion de ${beneficiaryFirstName} ${beneficiaryLastName}`,
```

Comment:
si j'en crois le titre, c'est un rappel non ? Dans cas est-ce qu'on peut ne nommer pour refleter ça ?

À moins que ça ne soit que le lien initial, et dans ce cas on peut revoir le wording de la phrase : `Immersion Facilitée - Bilan à compléter pour l'immersion de ...`

```suggestion
    AGENCY_ASSESSMENT_REMINDER: {
      niceName: "Prescripteurs - Rappel de Bilan à compléter",
      tags: ["notification prescripteur fin de l’immersion"],
      createEmailVariables: ({
        agencyLogoUrl,
        assessmentCreationLink,
        beneficiaryFirstName,
        beneficiaryLastName,
        businessName,
        conventionId,
        internshipKind,
      }) => ({
        subject: `Immersion Facilitée - Bilan non complété pour l'immersion de ${beneficiaryFirstName} ${beneficiaryLastName}`,
```

Reply:
J'ai revu les titres avec Magali, voir la PR à jour.